.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,108 @@
+name: 🐞 Bug report
```

i don't think we need any emojis
```suggestion
name: Bug report
```